ReadbasedAnalysis.wdl

```diff
@@ -1,37 +1,40 @@
+version 1.0
+
 import "ReadbasedAnalysisTasks.wdl" as tasks
 
 workflow ReadbasedAnalysis {
-    Boolean enabled_tools_gottcha2 = true
-    Boolean enabled_tools_kraken2 = true
-    Boolean enabled_tools_centrifuge = true
-    String db_gottcha2 = "/refdata/gottcha2/RefSeq-r223/gottcha_db.BAVFPt.species.fna"
-    String db_kraken2 = "/refdata/kraken2/"
-    String db_centrifuge = "/refdata/centrifuge/p_compressed"
-    Int cpu = 8
-    String input_file
-    String proj
-    String prefix=sub(proj, ":", "_")
-    Boolean? paired = false
-    String bbtools_container="microbiomedata/bbtools:38.96"
-    String? docker = "microbiomedata/nmdc_taxa_profilers:1.0.5"
+    input {        
+        Boolean enabled_tools_gottcha2 = true
+        Boolean enabled_tools_kraken2 = true
+        Boolean enabled_tools_centrifuge = true
+        String db_gottcha2 = "/refdata/gottcha2/RefSeq-r223/gottcha_db.BAVFPt.species.fna"
+        String db_kraken2 = "/refdata/kraken2/"
+        String db_centrifuge = "/refdata/centrifuge/p_compressed"
+        Int cpu = 8
+        String input_file
+        String proj
+        String prefix = sub(proj, ":", "_")
+        Boolean? paired = false
+        String bbtools_container = "microbiomedata/bbtools:38.96"
+        String docker = "microbiomedata/nmdc_taxa_profilers:1.0.5"
+    }
 
     call stage {
         input:
         container=bbtools_container,
         input_file=input_file
     }
 
-    if (enabled_tools_gottcha2 == true) {
+
```

Comment:
Is there a way to keep the if statements? I know these don't play as well with version 1.0. you use it in the reporting section so we should either remove all the boolean variables or keep them all.

Reply:
This is the recommendation instead of an else block
https://docs.openwdl.org/en/1.0.0/WDL/conditionals/

Reply:
cc @poeli

Reply:
I added the if statements back in